sandbox/basic.py

```diff
@@ -91,7 +91,7 @@ class BasicApp(App):
 
     def on_load(self):
         """Bind keys here."""
-        self.bind("tab", "toggle_class('#sidebar', '-active')")
+        self.bind("s", "toggle_class('#sidebar', '-active')")
```

Comment:
The app now uses tab. Which means we can't use tab elsewhere. Is this a reasonable sacrifice?

Reply:
The only use case of letting <tab> be used in the userland space of Textual would be for widgets that let people edit some code? In which case, the equivalent of a browser's textarea could take priority over the behaviour of "pressing tab" - but I reckon this would break accessibility. 
I don't know what the best practices are in terms of managing a11y and code editing?